### PR TITLE
Update outdated licence headers / Add missing licence headers

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | Update/Add licence headers
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI Green
| UI Tests          | not required
| Fixed issue or discussion?     | 
| Related PRs       | 
| Sponsor company   | PrestaShop SA

### CODE REVIEW

File: modules/index.php [hunk 1]
```diff
@@ -5,10 +5,10 @@
  *
  * NOTICE OF LICENSE
  *
- * This source file is subject to the Open Software License (OSL 3.0)
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
```
Comment (matthieu-rolland) [inline modules/index.php]: I have no idea if this change is legit 🤔

**Review by nicosomb — DISMISSED** [review]

**Review by nicosomb — DISMISSED** [review]

**Review by M0rgan01 — DISMISSED** [review]

**Review by boherm — DISMISSED** [review]

File: modules/index.php (hunk 1 above)
Comment (jolelievre) [reply modules/index.php]: I don't think so, this should use the Open source project license

File: src/PrestaShopBundle/Resources/views/Admin/Component/LegacyLayout/notifications_center.html.twig [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
```
Comment (FabienPapet) [inline src/PrestaShopBundle/Resources/views/Admin/Component/LegacyLayout/notifications_center.html.twig:24]: Duplicate licence here

**Review by FabienPapet — DISMISSED** [review]
There's some duplicate licences

File: src/PrestaShopBundle/Resources/views/Admin/Component/Layout/notifications_center.html.twig [hunk 3]
```diff
@@ -1,3 +1,27 @@
+{#**
```
Comment (Hlavtox) [inline src/PrestaShopBundle/Resources/views/Admin/Component/Layout/notifications_center.html.twig]: Duplicated license (also in other files)

**Review by Hlavtox — DISMISSED** [review]

**Comment by matthieu-rolland** [comment]
Double licence issues fixed, as well as weird licence in module/index.php

the CI is completely broken following a runner update on github, can we please merge this PR, it doesn't touch the codebase

**Review by nicosomb — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Pull request merged**